spec/scorecard_spec.rb

```diff
@@ -0,0 +1,117 @@
+require 'scorecard'
+
+describe ScoreCard do
+  let(:scorecard) { described_class.new }
+
+  it "should create a new scorecard" do
+    expect(scorecard).to be_a(ScoreCard)
+  end
+
+  it "should score a new game for a single player" do
+    expect(scorecard).to respond_to(:run).with(0).arguments
+  end
+
+  describe '#roll_1_input' do
+    it "should take a user input for their first roll" do
+      expect(scorecard).to receive(:gets).and_return("5\n")
+      scorecard.roll_1_input
+    end
+
+    it "should add the inputted score for roll 1 to frame_score" do
```

Comment:
A few of these tests are testing state rather than behaviour. [This ](https://github.com/makersacademy/skills-workshops/blob/master/practicals/testing/behaviour_not_state.md)is a practical about this topic 🙂